src/qml/FeatureForm.qml

```diff
@@ -434,6 +434,7 @@ Page {
             target: attributeEditorLoader.item
 
             function onValueChanged(value, isNull) {
+              console.log('ddd '+value)
```

Comment:
Do we keep that?

Reply:
Do we really have to get rid of my debugging crumbs? :wink: done.